polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -1004,7 +1038,7 @@ impl<T: Config> Pallet<T> {
 		loop {
 			// bail if we've iterated over all items.
 			if idx == 0 {
-				break
+				break;
```

Comment:
no unrelated changes please, `cargo +nightly-2023-11-01 fmt` it is.

Reply:
still having those semi. not sure why. will just manually delete them